src/view/com/notifications/FeedItem.tsx

```diff
@@ -207,23 +235,40 @@ export const FeedItem = observer(function ({
           />
           <ExpandedAuthorsList visible={isAuthorsExpanded} authors={authors} />
           <Text style={styles.meta}>
-            <TextLink
-              key={authors[0].href}
-              style={[pal.text, s.bold]}
-              href={authors[0].href}
-              text={sanitizeDisplayName(
-                authors[0].displayName || authors[0].handle,
-              )}
-            />
-            {authors.length > 1 ? (
-              <>
-                <Text style={[pal.text]}> and </Text>
-                <Text style={[pal.text, s.bold]}>
-                  {formatCount(authors.length - 1)}{' '}
-                  {pluralize(authors.length - 1, 'other')}
+            {listFormatNodes(
+              new Intl.ListFormat('en', {style: 'long', type: 'conjunction'}),
```

Comment:
This is giving me an error in iOS and Android; I'm pretty sure the RN runtime doesn't support it yet

Reply:
Ah. I see. That is why I had to slap the triple-slash on the top of the file to get TypeScript to be quiet.

&lt;sigh&gt; .. I would suggest my first version of this (it worked), but it was _pretty_ complex.

Would you be opposed to a poly-fill perhaps? (https://formatjs.io/docs/polyfills/intl-listformat/)

Reply:
Well I would, but see my other comment. I'm not sure I'm on board with the PR's intent unfortunately!